### DISCUSSION TITLE
Fan control patch

### DISCUSSION THREAD

**Discussion by flukejones** [dominiksalvet/asus-fan-control#107] [General]
So yeah, I'd been working on a fan control kernel patch [here](https://lkml.org/lkml/2021/10/23/249) which will upstream to the 5.16 kernel soon. So far I know it works on the G14 and G15 laptops, unsure of others though.

But I'm thinking this patch once merged could help serve as a base to support some of the laptops listed in your supported table. It would be good to standardise this and move away from acpi_call module.

**Comment by dominiksalvet** [comment]
Hmmm... what CPUs do those laptops have?
